ksqldb-serde/src/main/java/io/confluent/ksql/serde/KeyFormat.java

```diff
@@ -36,7 +37,7 @@
 public final class KeyFormat {
 
   private final FormatInfo format;
-  private final SerdeFeatures features;
+  private SerdeFeatures features;
```

Comment:
I thought this class was marked Immutable?

Reply:
um... yes it should be. I'm not sure why this changed (it might've been an accidental automated refactor)